### ISSUE TITLE
Move accept T&Cs into the actions panel

### ISSUE THREAD

**Issue by anthonydillon** [canonical-web-and-design/ubuntu.com#8395]
On all screens, it would be better to move the T&C checkbox into the action panel at the bottom to make it clean it is a required action. Particularly on small screens as you have to scroll to see it.
![image](https://user-images.githubusercontent.com/1413534/94693268-65b88280-032b-11eb-9b3e-4944744126d7.png)

**Comment by matthewpaulthomas** [comment]
I see that moving the checkbox to the commit area would make it more obvious. However:

- On those small screens it would add visual instability, because the commit area — and therefore also the main pane — would change in height when going from the payment-details view (which doesn’t have the checkbox) to the summary view (which does). Unless we also made the checkbox be present the whole time.

- In the summary view (and also the payment-details view, if the checkbox was present the whole time), the main pane would be shorter, so it would require more scrolling, so the payment generally would be a bit harder to understand and harder to scan for errors. (I just reported #8516 on a similar problem.)

- We’d need to find a new home for the spinner and progress text, which currently (on wide viewports) takes up the spot you’re proposing for the checkbox. (Centering the spinner over the dialog contents, as discussed in #8319, might solve that particular problem.)

That “Pay” is disabled conveys that _something_ you haven’t done yet is required. One tweak could be, if you still try clicking “Pay” before checking the checkbox, we could scroll the main pane to reveal the checkbox.
